app/src/main/java/com/example/t/MainActivity.kt

```diff
@@ -14,13 +14,12 @@ class MainActivity : AppCompatActivity() {
         setContentView(R.layout.activity_main)
 
         val btnButton : Button= findViewById(R.id.btnButton)
-        val Name : EditText = findViewById(R.id.Name)
         val Surname : EditText = findViewById(R.id.Surname)
         val DOB : EditText = findViewById(R.id.BOD)
```

Comment:
variable names should be camel case
Also avoid using abbreviations, rather say what it is dateOfBirthInputView